docs/install-and-run.md

```diff
@@ -64,6 +61,8 @@ For EarlGrey, we highly recommend [CocoaPods](https://cocoapods.org/) as the bes
   end
   ```
 
+    The earlgrey gem is supported with CocoaPods 1.0.0 and beyond. For earlier versions (or if you don't want a gem dependency) replace `require 'earlgrey'` with `load 'configure_earlgrey.rb'` and manually copy `configure_earlgrey.rb` from the gem into the same directory as the `Podfile`.
+    
```

Comment:
This looks good but I'd also replace the actual code (in line # 79) with `load 'configure_earlgrey.rb'` for v0.39 and below. thanks!